src/dlgPackageExporter.cpp

```diff
@@ -268,64 +317,230 @@ void dlgPackageExporter::slot_export_package()
         }
     }
 
-
-    int err = 0;
-    char buf[100];
-    zip* archive;
-    archive = zip_open(zipFile.toStdString().c_str(), ZIP_CREATE, &err);
-    if (err != 0) {
-        zip_error_to_str(buf, sizeof(buf), err, errno);
-        //FIXME: report error to user qDebug()<<"dp zip open error"<<zipFile<<buf;
-        close();
-        return;
-    }
-    QDir dir(tempDir);
-    QStringList contents = dir.entryList();
-    for (int i = 0; i < contents.size(); i++) {
-        QString fname = contents[i];
-        if (fname == "." || fname == "..") {
-            continue;
-        }
-        QString fullName = tempDir + "/" + contents[i];
-        struct zip_source* s = zip_source_file(archive, fullName.toStdString().c_str(), 0, 0);
-        if (s == nullptr) {
-            int sep = 0;
-            zip_error_get(archive, &err, &sep);
-            zip_error_to_str(buf, sizeof(buf), err, errno);
-            //FIXME: report error to userqDebug()<<"zip source error"<<fullName<<fname<<buf;
-        }
-        err = zip_file_add(archive, fname.toStdString().c_str(), s, ZIP_FL_OVERWRITE);
-        if (err == -1) {
-            int sep = 0;
-            zip_error_get(archive, &err, &sep);
-            zip_error_to_str(buf, sizeof(buf), err, errno);
-            //FIXME: report error to userqDebug()<<"added file error"<<fullName<<fname<<buf;
+    if (isOk) {
+        // zip error code:
+        int ze = 0;
+/*
+* Relevant options:
+* * ZIP_CREATE creates the archive if it does not exist.
+* * ZIP_TRUNCATE zaps any contents in a previously existing file.
+*/
+
+        // We previously only used ZIP_CREATE but without ZIP_TRUNCATE that
```

Comment:
Hard to grasp. Does "previously" refer to an older version of Mudlet, or to the proposed code here?

Reply:
Older version of Mudlet - will reword comment...

Reply:
Same question as below. Do we really want to keep comments on old Mudlet versions in source?

Reply:
Nope... I don't see any reason for it